packages/dawai-microservice/src/decorators/webservice.decorator.ts

```
@@ -0,0 +1,8 @@
+import { WebserviceDecoratorOptions } from '../decorator.options';
+import { metadataStorage } from './metadata.storage';
+
+export function webservice(options: WebserviceDecoratorOptions): ClassDecorator {
+  return (target: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)